llmstudio/engine/providers/provider.py

```diff
@@ -253,7 +264,9 @@ def join_chunks(self, chunks, request):
                     part = chunk.get("arguments", "")
```

Comment:
is this code ever running? how could we have an AzureRequest if we're in the scope of the condition of line 251? ie. `            elif isinstance(request, OpenAIRequest) or isinstance(
                request, VertexAIRequest
            ):`